README.md

```diff
@@ -286,22 +290,6 @@ $posts = PostFactory::randomSet(4); // array containing 4 "Post|Proxy" objects
 $posts = PostFactory::randomRange(0, 5); // array containing 0-5 "Post|Proxy" objects
 ```
 
-### Instantiate your `ModelFactory`
```

Comment:
Can I run `new PostFactory()` now? Or why was this removed?

Reply:
No, you still have to use `::new()`. However, you'll only use `new()` in complex cases, otherwise you use `Factory::createOne(...)` or `Factory::createMany(...)`. So I removed the main section about this and instead moved this into the complex code examples (line 331 and 361)